week-1/G-strings-methods/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
 
+var myName = "zareen khan";
+var namel = myName.length;
```

Comment:
`namel` isn't very clear as a variable name, is it? What else could you call it so it's clearer what this variable represents?

Reply:
Sure, but it would be clearer if the variable was renamed to something like `nameLength` :)